k8sClient.py

```diff
@@ -72,3 +72,15 @@ def ensure_oc_binary(self) -> None:
             lh.run("tar xf build/oc.tar.gz -C build")
             lh.run("rm build/oc.tar.gz")
         self.oc_bin = os.path.join(os.getcwd(), "build/oc")
+
+    def wait_for_mcp(self, mcp_name: str, resource: str = "resource"):
+        time.sleep(60)
+        iteration = 1
+        get_status_cmd = "get mcp sriov -o jsonpath='{.status.conditions[?(@.type==\"Updated\")].status}'"
+        while self.oc(get_status_cmd).out != "True":
```

Comment:
We have an infinite loop here. Could we use the iteration counter to somehow not be potentially stuck here forever.

Reply:
`logger.info(f"It took {minutes}m {seconds}s to for {resource} to be applied (attempt: {iteration})")` should be printed outside the loop.
But we can have in this place:
`logger.info(f"It took {minutes}m {seconds}s waiting for {resource}  (attempt: {iteration})")`